src/commands/TrackerCommands.ts

```diff
@@ -0,0 +1,196 @@
+import { Command } from "../interactions/interactionClasses";
+import {
+	ChatInputCommandInteraction,
+	SlashCommandBuilder,
+	SlashCommandSubcommandsOnlyBuilder,
+} from "discord.js";
+import { latest, logs, playtime, stats } from "../util/tracker/subCommands";
+import { userLast, userStats, userTop } from "../util/tracker/userCommands";
+import { gameLast, gameStats, gameTop } from "../util/tracker/gameCommands";
+import { addBlacklist, remBlacklist } from "../util/tracker/blacklistCommands";
+class Tracker extends Command {
+	constructor() {
+		super("tracker");
+	}
+
+	async execute(interaction: ChatInputCommandInteraction): Promise<void> {
+		const group = interaction.options.getSubcommandGroup();
+		const sub = interaction.options.getSubcommand();
+
+		switch (group + sub) {
+			case "userstats":
+				await userStats(interaction);
+				return;
+			case "userlast":
+				await userLast(interaction);
+				return;
+			case "usertop":
+				await userTop(interaction);
+				return;
+			case "gamestats":
+				await gameStats(interaction);
+				return;
+			case "gamelast":
+				await gameLast(interaction);
+				return;
+			case "gametop":
+				await gameTop(interaction);
+				return;
+			case "blacklistadd":
+				await addBlacklist(interaction);
+				return;
+			case "blacklistremove":
+				await remBlacklist(interaction);
+				return;
+			case "nullplaytime":
+				await playtime(interaction);
+				return;
+			case "nulllogs":
+				await logs(interaction);
+				return;
+			case "nulllast":
+				await latest(interaction);
+				return;
+			case "nullstats":
+				await stats(interaction);
+				return;
+		}
```

Comment:
Those are a LOT of commands. Could you reduce the amount a bit. You can maybe make one command out of multiple using some options

Reply:
Its the exact same amount as the current system (12 commands) just (I think) clearer to use. And I don't like the Idea to compact e.g. `game/user`-`stats/last/top` into less commands because it makes the commands messier to use. Because there would be more options which depending on what you what to do are just useless and confusing.

Reply:
The only commands I can imagine can be compacted without adding complexity and confusion are `blacklist add` and `blacklist remove` in form of `blacklist option:add game:VSCode`

Reply:
I like `game/user stats/last/top` and `blacklist add/remove`. Wouldn't make things more complicated. Just a few more if/switch statements in the code and you can keep everything you already have